src/main/java/com/chargerrobotics/commands/shooter/HoodAutoCommand.java

```diff
@@ -11,38 +11,40 @@
 import com.chargerrobotics.subsystems.LimelightSubsystem;
 import com.chargerrobotics.subsystems.ShooterHoodSubsystem;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class HoodAutoCommand extends CommandBase {
-  private final ShooterHoodSubsystem shooterSubsystem;
+  private static final Logger logger = LoggerFactory.getLogger(HoodAutoCommand.class);
+  private final ShooterHoodSubsystem shooterHoodSubsystem;
   private boolean isOpening;
   /**
    * Creates a new HoodAutoCommand.
    * 
    * Using the distance the robot is from the target, set the hood angle.
    * f(distance)
    */
-  public HoodAutoCommand(ShooterHoodSubsystem shooterSubsystem) {
-    this.shooterSubsystem = shooterSubsystem;
+  public HoodAutoCommand(ShooterHoodSubsystem shooterHoodSubsystem) {
+    this.shooterHoodSubsystem = shooterHoodSubsystem;
     // Use addRequirements() here to declare subsystem dependencies.
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    System.out.println("hood auto command init");
+    //If not calibrated, don't do anything
+    if (!shooterHoodSubsystem.getCalibrated()) return;
```

Comment:
Could you log an error here when this happens with something like:

`Cannot automatically move hood without calibrating it first!`

Reply:
Okay. We will be doing one more test next week, so I will make the changes then.